python/cuml/metrics/cluster/completeness_score.pyx

```diff
@@ -0,0 +1,95 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from cuml.common.handle cimport cumlHandle
+from libc.stdint cimport uintptr_t
+from cuml.metrics.cluster.mutual_info_utils import prepare_data
+import cuml.common.handle
+
+
+cdef extern from "cuml/metrics/metrics.hpp" namespace "ML::Metrics":
+    double completenessScore(const cumlHandle & handle, const int *y,
+                             const int *y_hat, const int n,
+                             const int lower_class_range,
+                             const int upper_class_range) except +
+
+
+def completeness_score(labels_true, labels_pred, handle=None):
+    """
+    Completeness metric of a cluster labeling given a ground truth.
+
+    A clustering result satisfies completeness if all the data points that are
+    members of a given class are elements of the same cluster.
+
+    This metric is independent of the absolute values of the labels:
+    a permutation of the class or cluster label values won’t change the score
+    value in any way.
+
+    This metric is not symmetric: switching label_true with label_pred will
+    return the homogeneity_score which will be different in general.
+
+    The labels in labels_pred and labels_true are assumed to be drawn from a
+    contiguous set (Ex: drawn from {2, 3, 4}, but not from {2, 4}). If your
+    set of labels looks like {2, 4}, convert them to something like {0, 1}.
+
+    Parameters
+    ----------
+    labels_pred : array-like (device or host) shape = (n_samples,)
+        The labels predicted by the model for the test dataset.
+        Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
+        ndarray, cuda array interface compliant array like CuPy
+    labels_true : array-like (device or host) shape = (n_samples,)
+        The ground truth labels (ints) of the test dataset.
+        Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
+        ndarray, cuda array interface compliant array like CuPy
+    handle : cuml.Handle
+        Specifies the cuml.handle that holds internal CUDA state for
+        computations in this model. Most importantly, this specifies the CUDA
+        stream that will be used for the model's computations, so users can
+        run different models concurrently in different streams by creating
+        handles in several streams.
+        If it is None, a new one is created.
+
+    Returns
+    -------
+    float
+      The completeness of the predicted labeling given the ground truth.
+      Score between 0.0 and 1.0. 1.0 stands for perfectly complete labeling.
+    """
+    handle = cuml.common.handle.Handle() if handle is None else handle
+    cdef cumlHandle *handle_ = <cumlHandle*> <size_t> handle.getHandle()
+
+    cdef uintptr_t preds_ptr
```

Comment:
You still need to define this pointers here so as I mentioned in the other point, its better to get the CumlArrays from `prepare_data` and then access them here with ary.ptr

Reply:
Agreed.